server/v1/controllers/auth/passwordChange.js

```diff
@@ -0,0 +1,45 @@
+const bcrypt = require('bcrypt-nodejs');
+const db = require('../../models');
+
+async function passwordChange(req, res) {
+  const { passConfirm, passNew, _id } = req.body;
+
+  if (!passConfirm || !passNew) {
+    return res.status(401).json('Password Confirm and/or Password New Missing');
+  }
+
+  if (passConfirm !== passNew) {
+    return res.status(401).json('Your passwords do not match!');
+  }
+
+  const user = await db.User.findOne({
```

Comment:
You don't need to find the user to update the password, all you have to do is `db.User.findOneAndUpdate`. You can also remove the validation check to see if it finds `user` below.